planning/autoware_obstacle_stop_planner/schema/adaptive_cruise_control.schema.json

```
@@ -0,0 +1,216 @@
+              "type": "number",
+              "description": "Maximum value of valid speed estimation results in speed estimation using a point cloud [m/s]",
+              "default": "20.0"
+            },
+            "valid_estimated_vel_min": {
+              "type": "number",
+              "description": "Minimum value of valid speed estimation results in speed estimation using a point cloud [m/s]",
+              "default": "-20.0"
+            },
+            "thresh_vel_to_stop": {
+              "type": "number",
+              "description": "Embed a stop line if the maximum speed calculated by ACC is lower than this speed [m/s]",
+              "default": "1.5"
+            },
+            "lowpass_gain_of_upper_velocity": {
+              "type": "number",
+              "description": "Lowpass-gain of upper velocity",
+              "default": "0.75"
+            },
+            "use_rough_velocity_estimation": {
+              "type": "boolean",
+              "description": "Use rough estimated velocity if the velocity estimation is failed (#### If this parameter is true, the vehicle may collide with the front car. Be careful. ####)",
+              "default": "false"
+            },
+            "rough_velocity_rate": {
+              "type": "number",
+              "description": "In the rough velocity estimation, the velocity of front car is estimated as self current velocity * this value",
+              "default": "0.9"
+            }
+          },
+          "required": [
+            "use_object_to_estimate_vel",
+            "use_pcl_to_estimate_vel",
+            "consider_obj_velocity",
+            "obstacle_velocity_thresh_to_start_acc",
+            "obstacle_velocity_thresh_to_stop_acc",
+            "emergency_stop_acceleration",
+            "emergency_stop_idling_time",
+            "min_dist_stop",
+            "obstacle_emergency_stop_acceleration",
+            "max_standard_acceleration",
+            "min_standard_acceleration",
+            "standard_idling_time",
+            "min_dist_standard",
+            "obstacle_min_standard_acceleration",
+            "margin_rate_to_change_vel",
+            "use_time_compensation_to_calc_distance",
+            "p_coefficient_positive",
+            "p_coefficient_negative",
+            "d_coefficient_positive",
+            "d_coefficient_negative",
+            "object_polygon_length_margin",
+            "object_polygon_width_margin",
+            "valid_estimated_vel_diff_time",
+            "valid_vel_que_time",
+            "valid_estimated_vel_max",
+            "valid_estimated_vel_min",
+            "thresh_vel_to_stop",
+            "lowpass_gain_of_upper_velocity",
+            "use_rough_velocity_estimation",
+            "rough_velocity_rate"
+          ],
+          "additionalProperties": false
+        },
+        "required": ["adaptive_cruise_control"],
+        "additionalProperties": false
+      }
+    }
+  },
```

There was a syntax error reported from CI: https://results.pre-commit.ci/run/github/429651616/1723125921.pCiSnW-4QVeMsk5QLEkaNQ


```suggestion
          "additionalProperties": false
        }
      },
      "required": ["adaptive_cruise_control"],
      "additionalProperties": false
    }
  },
```